react/ShelfItem.js

```diff
@@ -44,11 +52,17 @@ export default class ShelfItem extends Component {
     const { item, summary } = this.props
     const newSummary = assocPath(['name', 'tag'], 'h2', summary)
     return (
-      <ExtensionPoint
-        id="product-summary"
-        product={this.normalizeProduct(item)}
-        {...newSummary}
-      />
+        <ExtensionPoint
```

Comment:
why the indent? is your eslint environment configured correctly? :eyes:

Reply:
I don't even noticed it :/ I've had disabled 'autosave' and think I pasted this piece of code :/ But now turned on autosave again.